terraform/modules/aws/postgresql-setup/postgresql-setup.sh

```
@@ -507,32 +211,25 @@ echo '[CREATE] This should take 10-15 minutes. Progress will be shown below...'
 
 # Test ansible connection first
 echo '[TEST] Testing Ansible connectivity...'
-if ! ansible $NGINX_NODE_IP -i inventory.ini -m ping; then
+if ! timeout 30 ansible $NGINX_NODE_IP -i inventory.ini -m ping \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)